scripts/base/frameworks/intel/input.bro

```diff
@@ -24,7 +34,17 @@ event bro_init() &priority=5
 		{
 		for ( a_file in read_files )
 			{
-			Input::add_event([$source=a_file,
+			# Handle prefixing of the source file name. Note
+			# that this currently always uses the ASCII reader,
+			# so we know we're dealing with filenames.
+			local source = a_file;
+
+			# If we have a path prefix and the file doesn't
+			# already have an absolute path, prepend the prefix.
+			if ( |path_prefix| > 0 && sub_bytes(a_file, 0, 1) != "/" )
```

Comment:
Just a nit / informative comment: instead of `sub_bytes(a_file, 0, 1)`, string indexing like `a_file[0]` is supported and maybe easier to read/understand (maybe that didn't exist long ago, for sure, string-slicing, like `a_file[1:]` in Python, is a recent-ish addition).

Reply:
Oh my, that is handy to know ... thanks Jon.